karamel-common/src/main/java/se/kth/karamel/common/util/Settings.java

```diff
@@ -103,7 +103,7 @@ public static final String UNIQUE_GROUP_NAME(String provider, String clusterName
   }
 
   public static final String PREPARE_STORAGES_KEY = "karamel.prepare.storages";
-  public static final String PREPARE_STORAGES_DEFAULT = "false";
+  public static final String PREPARE_STORAGES_DEFAULT = "true";
```

Comment:
did you fix the prepare storages bash script because it used to be broken? And only worked with EBS instances. Will this break karamel for GCP?

Reply:
The prepare storage script is only used by EC2. I tried launching EC2 and GCP instances for both hops and hopsworks and it worked.